ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-release-4.10.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  openstack-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: openstack-installer
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.10"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.10"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-openstack-fips
+  interval: 48h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      FIPS_ENABLED: "true"
+      TEST_SUITE: experimental/reliability/minimal
+    pre:
+    - chain: ipi-openstack-pre
+    - ref: fips-check
+    workflow: openshift-e2e-openstack-ipi
+zz_generated_metadata:
+  branch: release-4.10
```

Comment:
the branch doesn't exist on that repo, I think this will fail. I'll probably create one for testing.

Reply:
What code do you want to test? It may make more sense to create a "variant" config, with a file called like `shiftstack-shiftstack-ci-master__4.10.yaml` which means "testing code from master but somehow this is a different config than we have for master branch itself".